src/main/java/xyz/vegaone/easytracking/monitor/ControllerMonitor.java

```diff
@@ -0,0 +1,18 @@
+package xyz.vegaone.easytracking.monitor;
+
+import lombok.extern.slf4j.Slf4j;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+@Slf4j
+public class ControllerMonitor {
```

Comment:
I think a better name for the class would be `ControllerLoggingAspect`. Class that implement AOP usually end in `Aspect`